bpf/cpu/cpu.bpf.c

```diff
@@ -162,6 +162,8 @@ u32 UNWIND_SAMPLES_COUNT = 7;
 
 /*================================ MAPS =====================================*/
 
+BPF_HASH(configuration, u8, u8, 1);
```

Comment:
@javierhonduco Do you know a more space-efficient way for these types of maps? I don't need the values; I want to check the existence of the keys.